xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -325,6 +325,14 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
      */
     var indentation = new LinkedHashMap<StringBuilder, String>()
     
+    /**
+     * Defines the execution environment that is used to execute binaries.
```

Comment:
Perhaps a more detailed comment would help. Suggest adding:

A given command may be directly executable on the host (NATIVE) or it may need to be executed
within a bash shell (BASH). On Unix-like machines, this is typically determined by the PATH variable
which could be different in these two environments.